pkgs/applications/science/chemistry/gwyddion/default.nix

```diff
@@ -10,7 +10,7 @@
   libxsltSupport ? true, libxslt ? null,
   fitsSupport ? true, cfitsio ? null,
   zlibSupport ? true, zlib ? null,
-  libuniqueSupport ? true, libunique ? null,
+  libuniqueSupport ? !stdenv.isDarwin, libunique ? null,
```

Comment:
If it builds on darwin, let's just add darwin to `libunique.meta.platforms`.

Reply:
I was a bit concerned about doing that without testing it myself first on OS X; I've just been able to check this for both libunique and gwyddion, and so added a commit to do this.  I'm not sure if I should create a separate pull request for libunique, however?

Reply:
These are small changes, so they are fine to do in one PR.